internal/controller/gcpinstall/inputs.go

```diff
@@ -94,6 +94,11 @@ func NewInputsController(params Params) *InputsController {
 			input = wrapinput.NewFreeForm()
 			input.Freeform.CharLimit = 0
 			input.Freeform.Placeholder = "project-id-1234"
+			if projects, err := gcp.GetProjectsFromEnvironment(); err == nil {
+				if projectsStr := strings.Join(projects, ","); len(projectsStr) > 0 {
+					input.Freeform.Placeholder = projectsStr
```

Comment:
Can it just be the first project in the list as the placeholder? Might be confusing to see multiple listed but you're only supposed to actually write one

Reply:
Done